Makefile

```diff
@@ -31,18 +30,21 @@ generate-proto: ## regenerate protos
 	@buf generate https://github.com/odpf/proton/archive/${PROTON_COMMIT}.zip#strip_components=1 --template buf.gen.yaml --path odpf/optimus
 	@echo " > protobuf compilation finished"
 
-unit-test:
-	go list ./... | grep -v -e third_party -e api/proto | xargs go test -count 5 -cover -race -timeout 1m -tags=unit_test
+unit-test-ci:
+	go test -count 5 -race -coverprofile coverage.txt -covermode=atomic -timeout 1m -tags=unit_test ./...
 
 smoke-test: build
 	@bash ./scripts/smoke-test.sh
 
-integration-test: 
-	go list ./... | grep -v -e third_party -e api/proto | xargs go test -count 1 -cover -race -timeout 1m
+integration-test:
+	go test -count 1 -cover -race -timeout 1m $(INTEGRATION_TEST_PATH)/...
```

Comment:
are we going to keep all integration tests in a specific directory or set of directories, I believe they can coexist with other unit tests so we need to run integration tests by configuring right tags.

Reply:
If we add tag for integration test, we need to annotate all files which are not for integration test with other tag, doc: https://pkg.go.dev/go/build

So using the run directive and rename all test as `TestIntegration`